### DISCUSSION TITLE
testflight插件切换地区无反应(Loon)

### DISCUSSION THREAD

**Discussion by yangchong99** [NSRingo/.github#17] [Q&A 问答] [answered]
iOS系统18.2b3
Loon版本(780)
清空了boxjs中testflight缓存，重新开关boxjs功能，单独测试了loon插件开关无效；开启boxjs开关，默认loon插件配置，改了国区无反应
![IMG_1210](https://github.com/user-attachments/assets/3f520ec3-f78c-431a-a349-f85bbf1ca3ca)
![IMG_1212](https://github.com/user-attachments/assets/26378b84-c8b1-409c-96e2-8d82618cdb6f)
![IMG_1211](https://github.com/user-attachments/assets/f4622f01-e1f7-4c7a-a055-381cbd3712e5)

**Comment by VirgilClyne** [comment, marked as answer]
https://nsringo.github.io/guide/box-js.html#%E6%B8%85%E9%99%A4%E6%9F%90%E4%B8%AA%E5%B9%B3%E5%8F%B0%E6%88%96%E7%B1%BB%E5%9E%8B%E7%9A%84%E8%AE%BE%E7%BD%AE%E6%88%96%E7%BC%93%E5%AD%98
